test/JDBC/input/openquery_upgrd-vu-verify.sql

```diff
@@ -1,5 +1,9 @@
 SELECT * FROM OPENQUERY(bbf_server, 'SELECT 123')
 GO
 
+-- should throw error as it takes more than 5 seconds to run
```

Comment:
Also check if sys.servers is showing correct query timeout value after upgrade

Reply:
done